ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml

```diff
@@ -6,6 +6,18 @@ images:
     RUN pip3 install pre-commit
   from: src
   to: openstack-operator-ci-pre-commit
+- build_args:
+  - name: KUTTL_VERSION
+    value: v0.15.0
+  dockerfile_literal: |
+    FROM openstack-operator-ci-build-deploy
+    ARG KUTTL_VERSION=v0.15.0
+    RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl
+    RUN chmod +x ./kubectl-kuttl
+    RUN mv ./kubectl-kuttl /usr/local/bin
+    RUN yum install -y skopeo
+  from: openstack-operator-ci-build-deploy
+  to: openstack-operator-ci-build-deploy-kuttl
```

Comment:
The job fails becuase the `openstack-operator-ci-build-deploy` job and images are not configured for this repo.

Reply:
Is there a ticket for that implementation? Was it missed?  Can I add it?

Reply:
I don't think so, AFAIK nobody worked much in horizon-operator CI. You can add it in this patch, taking this one as an example: https://github.com/openshift/release/pull/38395/files

Reply:
done, please let me know if it looks correct.

Reply:
looks ready to merge just needs the lgtm tag  @bshephar